release-notes-3.1.1.md

```diff
@@ -0,0 +1,53 @@
+# Release Notes
+
+This release makes no changes to the OpenAPI standard, however the formal specification document had multiple improvements, including:
+- clarity and expanded explanations including several new appendices with supplementary details
+- examples and additional documentation moved to https://learn.openapis.org
+- the HTML rendering of the specification at https://spec.openapis.org is now considered the authoritative version (formerly it was the Markdown source on GitHub)
+
+OpenAPI description writers should mark their OpenAPI descriptions with the version of OpenAPI specification they used to write their specification, updating where possible.
+
+Tool creators should also not expect changes, but it is recommended to check the list of changes below.
+
+## Clearer Definitions
+
+Introduce consistent language around OpenAPI Document/Description/Definition:
+- OpenAPI Description means the OpenAPI description of an API, whether it is in one or many files.
+- a document means a single file.
+- an "entry document" is where the OpenAPI description for an API starts; it may reference other documents.
+
+Improved language regarding schemas, explaining the difference between the OpenAPI schema, the schemas used within the OpenAPI schema, and the use of a non-authoritative JSON Schema to supplement the written spec.
+
+[3.1.1]Added guidance around use of schema dialects.
+
+## References
+
+Additional guidance for resolving references and parsing documents was added.
+Resolving component names, tags, and operationIds are clarified.
+[3.1.1]The adoption of JSON Schema in 3.1.x changed the parsing and referencing, and a new section was added to cover the changes in more depth than in 3.1.0.
```

Comment:
```suggestion
Additional guidance for resolving references and parsing documents was added.

Resolving component names, tags, and operationIds are clarified.

[3.1.1]The adoption of JSON Schema in 3.1.x changed the parsing and referencing, and a new section was added to cover the changes in more depth than in 3.1.0.
```

Reply:
These are a paragraph, the newlines are just for readability and easier editing. Splitting them up like this makes them one paragraph per sentence which might be all right, but is not at all what I was intending. I intentionally grouped related items together. Can we revisit these changes (sorry I didn't manage to review your reviews quicker!)

Reply:
Backed out change.